ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -264,12 +265,13 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
         };
         const thresholdName = step.name ?? `Threshold ${index + 1}`;
 
-        // generates array of [time, step.value] where time ranges from timescale.startMs to timescale.endMs with an interval of 30s
+        // Generates array of [time, step.value] where time ranges from timescale.startMs to timescale.endMs with an interval of 15s
         const thresholdTimeValueTuple: TimeSeriesValueTuple[] = [];
         let currentTimestamp = timeScale.startMs;
         while (currentTimestamp <= timeScale.endMs) {
           thresholdTimeValueTuple.push([currentTimestamp, stepOption.value]);
-          currentTimestamp += 1000 * 30;
+          // Used to plot fake thresholds datapoints so correct nearby threshold series shows in tooltip without flicker
+          currentTimestamp += 1000 * THRESHOLD_PLOT_INTERVAL;
```

Comment:
@eunicorn I reduced this interval from 30s to 15s since moving your cursor horizontally close to a threshold line had this flicker before (probably should have made a separate PR but noticed it when testing datapoint hover state): 

https://github.com/perses/perses/assets/9369625/1ac02a71-3d34-4da5-9723-b617b23b337a